docs/user_guide/mitigation.rst

```diff
@@ -611,8 +611,8 @@ Group :code:`"a"` has an average loss of :math:`0.05`, while group
     >>> y_true             = np.array([0.3, 0.5, 0.1, 1.0])
     >>> y_pred             = np.array([0.3, 0.6, 0.6, 0.5])
     >>> sensitive_features = np.array(["a", "a", "b", "b"])
-    >>> mae_frame = MetricFrame(mean_absolute_error,
-    ...                         y_true, y_pred,
+    >>> mae_frame = MetricFrame(metrics=mean_absolute_error,
+    ...                         y_true=y_true, y_pred=y_pred,
```

Comment:
Small thing, but if an argument list is being split over multiple lines, I usually like to keep one argument per line (opinions on this may differ, and I'm not wholly consistent either)

Reply:
i can do this... this reminds me... do we want to switch to black over flake8? pros/cons?

Reply:
to me, black does the formatting, flake8 tests the compliance.